### PR TITLE
Add tests for MixServerHandler

### PR DESCRIPTION
**Pull request by maropu**

### CODE REVIEW

File: mixserv/src/main/java/hivemall/mix/server/MixServerHandler.java [hunk 1]
```diff
@@ -133,22 +146,21 @@ private void mix(final ChannelHandlerContext ctx, final MixMessage requestMsg, f
...
-                if(diffClock >= syncThreshold) {// sync model if clock DIFF is above threshold
+                if(diffClock >= syncThreshold) { // sync model if clock DIFF is above threshold
                     float averagedWeight = partial.getWeight(scale);
                     float meanCovar = partial.getCovariance(scale);
                     short globalClock = partial.getClock();
-                    responseMsg = new MixMessage(event, feature, averagedWeight, meanCovar, globalClock, 0 /* deltaUpdates */);
+                    responseMsg = new MixMessage(
+                            event, feature, averagedWeight, meanCovar, globalClock,
+                            0 /* deltaUpdates */);
+                    ctx.writeAndFlush(responseMsg);
```
Comment (myui) [inline mixserv/src/main/java/hivemall/mix/server/MixServerHandler.java]: No need to change `mix()`.

`ctx.writeAndFlush(responseMsg)` should be called outside `partial.lock()`.
The original code that send a response in `mix()` is better.

Comment (maropu) [inline mixserv/src/main/java/hivemall/mix/server/MixServerHandler.java]: This is my bad for `writeAndFlush`, so I'll fix it.

Why `SessionObject session = getSession(msg)` and `PartialResult partial = getPartialResult(msg, session)` is outside `mix`? If these sentences move to `mix`, argument `session` is not necessary anymore.

File: mixserv/src/main/java/hivemall/mix/server/MixServerHandler.java [hunk 2]
```diff
@@ -123,7 +133,13 @@ private void mix(final ChannelHandlerContext ctx, final MixMessage requestMsg, f
         final int deltaUpdates = requestMsg.getDeltaUpdates();
         final boolean cancelRequest = requestMsg.isCancelRequest();
 
-        MixMessage responseMsg = null;
+        if (deltaUpdates <= 0)
```
Comment (myui) [inline mixserv/src/main/java/hivemall/mix/server/MixServerHandler.java]: Please use this modification only to mix().
BTW, put `{` and `}` for every conditional branch.

**Comment by myui** [comment]
Merged w/ some modifications in this commit.
https://github.com/myui/hivemall/commit/4f0e07c397ff8790634540f4141c028a244ebb4b

**Comment by maropu** [comment]
Thanks for the merging.

**Pull request closed without merging**